src/components/MessageBubble.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../styles/theme.style.js';
+import { StyleSheet, Text, View, Image, TouchableOpacity, TextInputComponent, } from 'react-native';
+import TextInputContainer from '../containers/TextInputContainer';
+import { TextInput } from 'react-native-gesture-handler';
+
+
+class MessageBubble extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        isAuthor: true,
+        isSelectable: true,
+      };
+    }
+  
+    render() {
+      return (
+        <MessageBubbleContainer isAuthor={this.state.isAuthor} backgroundColor={this.props.labelColor} isSelectable={this.props.isSelectable}>
+           <WrittenMessage
+           placeholder='Message'
+       placeholderTextColor={"#D8D8D8"}
+       multiline={true}
+     />
+          <TextInputContainer placeholder='' isConfirmed={false}>
+          </TextInputContainer>
+        </MessageBubbleContainer>
+      ); 
+    }
+  }
+  
+const MessageBubbleContainer = styled.View`
+background-color: ${props => props.isAuthor ? theme.COLOR_ORANGE : (props.backgroundColor ? props.backgroundColor :  theme. COLOR_LIGHT_GRAY)};
```

Comment:
- Remove the space after theme. COLOR_LIGHT_GRAY, as I don't think it would call the variable
- You don't have to take in backgroundColor, as for now we are putting it orange!
  - For simplicity, remove the props.backgroundColor and just keep the orange color in the else statement